README.md

```diff
@@ -30,12 +30,12 @@ To create/retrieve a key the user calls:
 
 Create/retrieve key:
 
-    if err, key1 := tags.CreateKeyString("keyNameID1"); err != nil {
+    if key1, err := tags.CreateKeyString("keyNameID1"); err != nil {
```

Comment:
FYI: github support code blocks: https://help.github.com/articles/creating-and-highlighting-code-blocks/

Reply:
Cool. Thanks for the tip. Updated the README.md to use that.

Reply:
You need to remove the spaces before each line.
Otherwise github thinks ` ```go ` is part of the code block and shows it verbose.

Reply:
ok